java/code/src/com/suse/manager/reactor/messaging/test/JobReturnEventMessageActionTest.java

```diff
@@ -613,15 +613,16 @@ public void testPackagesProfileUpdateWithCaaSPSystemLocked() throws Exception {
         action.addServerAction(ActionFactoryTest.createServerAction(minion, action));
         HibernateFactory.getSession().flush();
         // Setup an event message from file contents
-        Optional<JobReturnEvent> event = JobReturnEvent.parse(
-                getJobReturnEvent("packages.profileupdate.caasp.json", action.getId()));
+        Optional<JobReturnEvent> event = null;
+
+        event = JobReturnEvent.parse(
+                getJobReturnEvent("packages.profileupdate.caasp-node.json", action.getId()));
```

Comment:
Why is the assignment separated?

Reply:
It was a leftover from debugging. Fixed, thanks!